sdk/src/org.graalvm.home/src/org/graalvm/home/HomeFinder.java

```diff
@@ -108,10 +106,3 @@ public static HomeFinder getInstance() {
         }
     }
 }
-
-class HomeFinderFeature implements Feature {
```

Comment:
Maybe we should keep the class around, even if just empty, to not break potential external usages of `--features=org.graalvm.home.HomeFinderFeature`? I'm not sure.

Reply:
Keeping around an empty feature class would be missleading.